app/views/layouts/application.html.erb

```diff
@@ -9,9 +9,21 @@
 </head>
 <body>
   <div class="container">
-      <ul class="nav nav-tabs">
-        <li> <%= link_to "Home", welcome_index_path %> </li>
+      <ul class="nav nav-tabs" style="border-bottom:none;">
+        <li> <%= link_to "Blocipedia", welcome_index_path %> </li>
         <li> <%= link_to "About", welcome_about_path %> </li>
+<!-- 1 -->
```

Comment:
You really should remove these kinds of comments. They only make sense when looking at the curriculum and don't belong in your code.